CppHeaderParser/CppHeaderParser.py

```diff
@@ -2375,13 +2375,30 @@ def _evaluate_method_stack(self):
         self.stack = []
         self.stmtTokens = []
 
+    def _function_point_typedef_parse(self, stack):
+        idx = stack.index("typedef")
+        expression = "".join(stack[idx + 1 :])
+        pattern = re.compile(r".*?\(.*?\*(.*?)\).*?\(.*?\).*?")
```

Comment:
Can you lift this out to be a global so it's not computed over and over again?

Reply:
yeah, it's a good suggestion and already done.